### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by costa2400**
enhancements have been made to the documentation of the Sylvia Framework, focusing on clarity and comprehensibility. The primary changes include the simplification of technical jargon, restructuring of complex explanations, and the addition of detailed commentary around code examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,42 @@
...
-have to think about message structure, how their API is (de)serialized, or how
-to handle message dispatching. Instead, the API of your contract is a set of
-traits you implement on your SC type. The framework generates things like entry
-point structures, functions dispatching the messages, or even helpers for multitest.
-It allows for better control of interfaces, including validating their completeness
-in compile time.
+The [CosmWasm](https://cosmwasm.com/) ecosystem lays the foundational blocks for smart contracts:
+[cosmwasm-std](https://crates.io/crates/cosmwasm-std) for essential CW bindings, [cw-storage-plus](https://crates.io/crates/cw-storage-plus) for simplified state management, and [cw-multi-test](https://crates.io/crates/cw-multi-test) for effiecient testing. Built atop these, the Sylvia framework liberates you from the intricacies of the message structures, , API serialization, and message dispatching. Your contract's API becomes a suite of traits implemented on your SC type, while Sylvia seamlessly generates entry point structures and message dispatching functions, alongside multi-test helpers for an enhanced development experience.
```
Comment (jawoznia) [inline README.md:14]:
```suggestion
[cosmwasm-std](https://crates.io/crates/cosmwasm-std) for essential CW bindings, [cw-storage-plus](https://crates.io/crates/cw-storage-plus) for simplified state management, and [cw-multi-test](https://crates.io/crates/cw-multi-test) for effiecient testing. Built atop these, the Sylvia framework liberates you from the intricacies of the message structures, API serialization, and message dispatching. Your contract's API becomes a suite of traits implemented on your SC type, while Sylvia seamlessly generates entry point structures and message dispatching functions, alongside multi-test helpers for an enhanced development experience.
```

**Review by jawoznia — APPROVED** [review]
One nitpick, otherwise LGTM

File: README.md [hunk 2]
```diff
@@ -730,6 +688,14 @@ recommendation is to add an additional `sylvia` entry with `mt` enabled in the
 `dev-dependencies`, and also add the `mt` feature on your contract, which enables
 mt utilities in other contract tests. An example `Cargo.toml`:
 
+Sylvia enhances your smart contract testing experince by providing specialized testing utilities, conveniently accesible under the `mt`(multi-test) feature flag. This feature is instrumental in creating a robus and reliable testing enviroment for your contracts.
+
+### Setting Up the Multi-test feature
+
+To leverage Sylvia's multi-test capabilities, you must enable the `mt` feature. However, it's crucial to note that this feature should not be active when building your contract for the Wasm target. This precaution is due to certain dependencies in the `mt` feature that are incompatible with Wasm builds. To circumvent this, it's recommended to include Sulvia with the `mt` feature enabled in your `dev-depencies`. This setup allows you to utilize the multi-test utilities in your contract tests without affecting the Wasm build.
```
Comment (jawoznia) [inline README.md:695]:
```suggestion
To leverage Sylvia's multi-test capabilities, you must enable the `mt` feature. However, it's crucial to note that this feature should not be active when building your contract for the Wasm target. This precaution is due to certain dependencies in the `mt` feature that are incompatible with Wasm builds. To circumvent this, it's recommended to include Sylvia with the `mt` feature enabled in your `dev-dependencies`. This setup allows you to utilize the multi-test utilities in your contract tests without affecting the Wasm build.
```

**Comment by kulikthebird** [comment]
We need to update the latest version of README, there should be a new PR created for that

**Pull request closed without merging**